ember-toucan-styles/src/services/theme-manager.ts

```diff
@@ -60,11 +81,11 @@ export default class ThemeManagerService extends Service {
    *
    * @param {THEMES} theme
    */
-  forceSelectTheme(theme: Theme) {
+  forceSelectTheme(theme: Theme, doc = document) {
```

Comment:
how come `doc` needs to be passed everywhere? 
can we only reference it in `selectTheme`, and then assign it once, maybe in `setup`?
(this ends up aligning with the current readme about how to initialize the theme manager)